src/lib/pages/backoffice/Acquisition/Order/OrderEditor/OrderForm/OrderForm.js

```diff
@@ -100,10 +104,25 @@ export class OrderForm extends Component {
     return orderApi.update(pid, data);
   };
 
-  successCallback = response => {
-    goTo(
-      AcquisitionRoutes.orderDetailsFor(getIn(response, 'data.metadata.pid'))
+  successCallback = async response => {
+    const order = getIn(response, 'data');
+    const documentRequestPid = _get(
+      this.props,
+      'data.documentRequestPid',
+      null
     );
+    if (documentRequestPid) {
+      await documentRequestApi.addProvider(documentRequestPid, {
+        physical_item_provider: {
+          pid: order.pid,
```

Comment:
`order.metadata.pid` (verify if it works please), we try to abandon the `record.pid` because it is inconsistent with the objects structures, we introduced it at some point in the serializers, but it is confusing IMO